### PR TITLE
Suparna_Assignment_24_10Min

### PR DESCRIPTION
**Pull request by SuparnaNikam**
phonenovalidation

### CODE REVIEW

**Review by VireshJJun20 — APPROVED** [review]
Code looks good to me.

File: src/suparnaNJune20/Assignment_24/PhoneNoValidation.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+void validPhoneNo(String phoneNo) {
+		switch (phoneNo.length()) {
+		case 14:
+			if (phoneNo.charAt(0) == '+' && phoneNo.charAt(1) == '9' && phoneNo.charAt(2) == '1'
+					&& phoneNo.charAt(3) == ' '
+					&& (phoneNo.charAt(4) == '9' || phoneNo.charAt(4) == '8' || phoneNo.charAt(4) == '7')
+					&& (Character.isDigit(phoneNo.charAt(5))) && (Character.isDigit(phoneNo.charAt(6)))
+					&& (Character.isDigit(phoneNo.charAt(7))) && (Character.isDigit(phoneNo.charAt(8)))
+					&& (Character.isDigit(phoneNo.charAt(9))) && (Character.isDigit(phoneNo.charAt(10)))
+					&& (Character.isDigit(phoneNo.charAt(11))) && (Character.isDigit(phoneNo.charAt(12)))
```
Comment (KrishnaKTechnocredits) [inline src/suparnaNJune20/Assignment_24/PhoneNoValidation.java:26]: You should check +91 character after all numbers are digit or not. in one shot.

File: src/suparnaNJune20/Assignment_24/PhoneNoValidation.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+			break;
+		case 11:
+			if (phoneNo.charAt(0) == '0'
+					&& (phoneNo.charAt(1) == '9' || phoneNo.charAt(1) == '8' || phoneNo.charAt(1) == '7')
+					&& (Character.isDigit(phoneNo.charAt(2))) && (Character.isDigit(phoneNo.charAt(3)))
+					&& (Character.isDigit(phoneNo.charAt(4))) && (Character.isDigit(phoneNo.charAt(5)))
+					&& (Character.isDigit(phoneNo.charAt(6))) && (Character.isDigit(phoneNo.charAt(7)))
+					&& (Character.isDigit(phoneNo.charAt(8))) && (Character.isDigit(phoneNo.charAt(9)))
+					&& (Character.isDigit(phoneNo.charAt(10))))
```
Comment (KrishnaKTechnocredits) [inline src/suparnaNJune20/Assignment_24/PhoneNoValidation.java:40]: Same applied here.

File: src/suparnaNJune20/Assignment_24/PhoneNoValidation.java (hunk 1 above)
Comment (SuparnaNikam) [reply src/suparnaNJune20/Assignment_24/PhoneNoValidation.java:26]: Replaced with function which check string char is digit or not .

File: src/suparnaNJune20/Assignment_24/PhoneNoValidation.java (hunk 2 above)
Comment (SuparnaNikam) [reply src/suparnaNJune20/Assignment_24/PhoneNoValidation.java:40]: Replaced with function which check string char is digit or not .

**Review by harshvegada — APPROVED** [review]
@SuparnaNJune20 your changes is not as the krishna's comments.

**Pull request merged**